script.js

```diff
@@ -1,6 +1,52 @@
 const burgerBtn = document.querySelector('.ham0');
 const navMenu = document.querySelector('.nav_desktop');
 const crossBtn = document.querySelector('.crossBtn');
+const speakerInfo = document.querySelector('.speaker_info');
+
+const data = [{
+  speakerImage: 'images/rolex_ceo.jpg',
+  name: 'Jean-Frédéric Dufourr',
+  alt: 'Jean-Frédéric Dufour image',
+  profession: "Citizen Watch's President & CEO",
+  achievements: 'Jean-Frédéric Dufourr is an American business executive who has been the chief executive officer of Rolex Watch Inc. since 2020.',
+},
+{
```

Comment:
- [ ] You did great here too. For readability purposes, could you please reformat the array of objects to look something like this:point_down:
```
const data = [
  {
    speakerImage: '......',
    name: "...",
  },
  {
    ...
  }
];
```
You get the idea.